domain/miningmanager/mempool/check_transaction_standard.go

```diff
@@ -129,7 +128,7 @@ func (mp *mempool) IsTransactionOutputDust(output *externalapi.DomainTransaction
 	// The most common scripts are pay-to-pubkey, and as per the above
 	// breakdown, the minimum size of a p2pk input script is 148 bytes. So
 	// that figure is used.
-	totalSerializedSize := estimatedsize.TransactionOutputEstimatedSerializedSize(output) + 148
+	totalSerializedSize := transactionvalidator.TransactionOutputEstimatedSerializedSize(output) + 148
```

Comment:
Don't we prefer to just change it to use mass?

Reply:
So just to be clear (I think I agree, but want to make sure we're on the same page):
We remove maximumStandardSignatureScriptSize completely, because we SignatureScriptSize is embedded into mass, and the limit comes from there?